content/en/docs/contribute/style/style-guide.md

```diff
@@ -244,7 +244,7 @@ kind: Pod
 
 ## Kubernetes.io word list
 
-A list of Kubernetes-specific terms and words to be used consistently across the site.
+A list of Kubernetes-specific terms and words to be used consistently in text across the site.
```

Comment:
Hi @natereid72 . Thanks for opening this pull request.
I think the original sentence is clear.  Is there a specific style question in the docs?

Reply:
> I think the original sentence is clear. Is there a specific style question in the docs?

Ok by me either way. It's just weird to me that it says 'to be used consistently across the site' stating that Kubernetes should always be capitalized when the logo itself is lower case. Not losing sleep over it though.

Reply:
Hi @natereid72 – I agree with @kbhawkey that the original sentence is clear. Given Kubernetes is a product, it being capitalized is an important distinction, as opposed to the logo which is a style choice. Thanks!

Reply:
> Kubernetes is a product

It is? I thought Kubernetes was an open source project. [Docs - What is Kubernetes](https://kubernetes.io/docs/concepts/overview/what-is-kubernetes/).

Whether it's a product or a project, the statement that all occurrences should be capitalized is ironic in this case at a minimum.  Adding those two words to the statement makes it clear. The statement is about a style choice in the text as well.

Reply:
@natalisucks ok, sounds like a reasonable concern. There must be some note that can be added to the style guide, in some way that doesn't create that potential issue. In this example of diagrams that are potentially not `in text`, then how does that differ from a logo? 

Net/net, it's difficult to say `all` when the logo that is injected on every page breaks the rule.

Reply:
>In this example of diagrams that are potentially not in text, then how does that differ from a logo?

@natereid72 [This page](https://kubernetes.io/docs/concepts/overview/components/) contains a diagram that follows this guideline as a way to address the difference. I hope that brings some clarity – let me know if not. I know it seems like a very small detail, but even labels in diagrams like this need to follow the same guidelines.

Perhaps adding an example of what we mean by `all` here could be a happy medium. For example: "A list of Kubernetes-specific terms and words to be used consistently across the site, including text and diagrams". Or something similar?

Reply:
@natalisucks After taking some time to think about this, and coming back to it, adding an explanation for what we mean by `all` is inline with my initial thought. But then it seems a bit like kicking the can further down the road. 

And it gets me to an either/or sort of position. Either the Kubernetes logo changes to conform with this style policy, or the style policy changes to address the fact that the logo is kubernetes.

Why go through contortions to explain away that everyone should write `Kubernetes` instead of `kubernetes` on a page that always starts with `kubernetes` at the top? Just let everyone feel free to write `kubermetes`? Even further into this question, why not just recommend in style that everyone write `k8s`? k8s is born from i18n that was also not capitalized, because no one wanted to continuously type `Internationalization`. If k8s serves that purpose (albeit nowhere near as irritating as typing out Internationalization), then why not say `use k8s`?